backend/src/services/cars/carsSearch.service.ts

```diff
@@ -0,0 +1,135 @@
+import { CarsSearchParams } from '@autoline/shared';
```

Comment:
All the files should be named in `kebab-case`. Please rename the files named in `snake_case` and `camelCase`